pkg/agent/metrics/prometheus.go

```diff
@@ -112,4 +130,18 @@ func InitializePrometheusMetrics() {
 	if err := legacyregistry.Register(OVSFlowCount); err != nil {
 		klog.Error("Failed to register antrea_agent_ovs_flow_count with Prometheus")
 	}
+
+	if err := legacyregistry.Register(OVSFlowOpsErrorCount); err != nil {
+		klog.Error("Failed to register antrea_agent_ovs_flow_ops_error_count with Prometheus")
+	}
+	OVSFlowOpsErrorCount.WithLabelValues("add")
+	OVSFlowOpsErrorCount.WithLabelValues("modify")
+	OVSFlowOpsErrorCount.WithLabelValues("delete")
+
+	if err := legacyregistry.Register(OVSFlowOpsDuration); err != nil {
+		klog.Error("Failed to register antrea_agent_ovs_flow_ops_duration_milliseconds with Prometheus")
+	}
+	OVSFlowOpsDuration.WithLabelValues("add")
```

Comment:
same question as above?

Reply:
Commented on above. I'll change this in the same way.